src/interfaces/cli.js

```diff
@@ -15,8 +15,7 @@ class CliClient {
   }
 
   sendMessage(channel, message) {
-    console.log(`[${this.nick} => ${channel}]\n${message}`);
-    console.log(`[${channel} => ${this.nick}]`);
+    console.log(message);
```

Comment:
What would be the problem here? Do you mean the reference to `this.nick` may have mutated? Because channel and message certainly will not have changed...